www/js/splash/notifScheduler.ts

```diff
@@ -87,22 +87,23 @@ export const getScheduledNotifs = function (isScheduling: boolean, scheduledProm
       );
       logDebug('requesting fetch while still actively scheduling, waiting on scheduledPromise');
       scheduledPromise.then(() => {
-        getNotifs().then((notifs: object[]) => {
+        getNotifs().then((notifs) => {
```

Comment:
Since I added a type to the promised returned value of `getNotifs` (`ScheduledNotif[]`), Typescript infers the type here as such.
So if you mouse over `notifs` in VSCode, it is identified as a `ScheduledNotif[]`, which is more descriptive than `object[]` anyway
<img width="445" alt="image" src="https://github.com/e-mission/e-mission-phone/assets/15843932/47e6b400-cd9a-4fcd-a8b4-85aa6b25db3f">